.gitignore

```diff
@@ -0,0 +1,6 @@
+﻿################################################################################
```

Comment:
This .gitignore file has a very specific file which probably isn't visible to Git anyway.  But a better way to ignore this is to ignore all files in ModelViewer/.vs/